lua/main_config.lua

```diff
@@ -36,19 +35,25 @@ function gatekeeper_init()
 	local gt_conf
 
 	if gatekeeper_server == true then
-		-- n_lcores + 2 on same NUMA: for GK-GT Unit and Solicitor.
+		-- n_lcores + 1 on same NUMA: for GK-GT Unit.
 		local n_lcores = 2
 		local gk_lcores =
 			staticlib.alloc_lcores_from_same_numa(numa_table,
-				n_lcores + 2)
-		local sol_lcore = table.remove(gk_lcores)
+				n_lcores + 1)
 		local ggu_lcore = table.remove(gk_lcores)
 
+		local n_sol_lcores = 1
+		local sol_lcores =
+			staticlib.alloc_lcores_from_same_numa(numa_table,
```

Comment:
Once pull request #391 is merged, we need to spread the SOL blocks among the NUMA nodes using those functions.